server/routes/students.js

```diff
@@ -33,6 +34,29 @@ router.post('/', async (req, res, next) => {
         const emailVerified = false
         const approved = false
         const verificationToken = crypto({length: 16});
+
+        // find or create College
```

Comment:
We probably don't need to change this post end-point as students would not add colleges to their shortlist when they sign up